.appveyor.yml

```diff
@@ -1,17 +1,10 @@
-environment:
-
-  image: Visual Studio 2017
+image: Visual Studio 2019
 
+environment:
   matrix:
     - PYTHON: "C:\\Python36"
-      PYTHON_VERSION: "3.6.x"
-      PYTHON_ARCH: "32"
```

Comment:
I'm not sure these are neccessary. possibly `PYTHON_ARCH` is, but I don't think so. `PYTHON_VERSION` is redundant with `PYTHON`